script.js

```diff
@@ -1,62 +1,88 @@
-const myTitle = document.getElementById('input');
-const myAuthor = document.getElementById('input1');
-const myAdd = document.getElementById('bttn');
-const myList = document.getElementById('list-holder');
-const Error = document.getElementById('error');
-
-const BOOKS_KEY = 'awesome-books';
-let books = JSON.parse(localStorage.getItem(BOOKS_KEY)) || [];
-function saveBooks() {
-  localStorage.setItem(BOOKS_KEY, JSON.stringify(books));
+/* eslint-disable */ 
+class Book {
```

Comment:
- [ ] Please remove any eslint disablement lines, and kindly solve any linters errors yourself